samples/user-interface/haptics/README.md

```diff
@@ -0,0 +1,115 @@
+Haptic Sampler
+===========================
+
+Android provides wide access to the device vibrator hardware so you can play different
+vibration effects and patterns from your app. 
+
+Haptics can enrich your app experience by attracting
+the user's attention when it’s required, providing useful feedback to their interactions, and
+augmenting audio-visual content with tactile sensations.
+
+The sample is built on top of [Jetpack Compose](https://developer.android.com/jetpack/compose) and requires a recent version of [Android Studio](https://developer.android.com/studio). 
+
+It uses the Gradle build system. To build this project, clone the repo and use the "gradlew build" command or use "Import Project" in Android Studio (***File > New > Import Project***).
+
+> Important Note: Many of the primitives used in this sample are not yet widely available
+on Android phones.The app will notify you if the device you are using does not
+support the example as intended. 
+
+# Home screen & examples
+
+## Home screen
```

Comment:
Since now the samples are separate, rephrase this

Reply:
Done.